assets/stylesheets/bootstrap/_mixins.scss

```diff
@@ -6,6 +6,7 @@
 @import "mixins/opacity";
 @import "mixins/image";
 @import "mixins/labels";
+@import "mixins/reset-filter";
```

Comment:
reset-filter I believe is 100% IE/Microsoft and no longer needed or supported , am I not right ?